load.php

```diff
@@ -5,6 +5,8 @@
 use function Altis\get_environment_architecture;
 use function Altis\register_module;
 
+require_once __DIR__ . '/inc/namespace.php';
```

Comment:
Think this should be in the Composer autoload.files instead

Reply:
I can change it, but I was going off of convention from the other packages